SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -34,10 +34,10 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
     var isSpinning: Bool
     var incomingAlertController: UIAlertController?
 
-    var callInvite: TVOCallInvite?
-    var call: TVOCall?
     var callKitCompletionCallback: ((Bool)->Swift.Void?)? = nil
     var audioDevice: TVODefaultAudioDevice = TVODefaultAudioDevice()
+    var activeCallInvites: [TVOCallInvite]? = []
+    var activeCalls: [TVOCall]? = []
```

Comment:
Can these arrays be non-optionals?  So that you don't have to force unwrapping them in the code where they are used.